qa/L0_pyprof_data/test.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+ # Copyright (c) 2020, NVIDIA CORPORATION. All rights reserved.
+ #
+ # Licensed under the Apache License, Version 2.0 (the "License");
+ # you may not use this file except in compliance with the License.
+ # You may obtain a copy of the License at
+ #
+ #     http://www.apache.org/licenses/LICENSE-2.0
+ # 
+ # Unless required by applicable law or agreed to in writing, software
+ # distributed under the License is distributed on an "AS IS" BASIS,
+ # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ # See the License for the specific language governing permissions and
+ # limitations under the License.
+
+TEST_LOG="./data.log"
+
+
+apt-get update && \
+    apt-get install -y --no-install-recommends python
+
+rm -f $TEST_LOG
+RET=0
+
+./test_pyprof_data.py > $TEST_LOG 2>&1
+if [ $? -ne 0 ]; then
```

Comment:
Does this output any files? If so, the should be a check to see if the files exist and not zero.

Reply:
The file output is just a cat of the test output. 
I thought it was standard practice to do that in our testing. I don't rely on the presence of the log file to pass. I just want the output of the run in the CI if it fails to aid in debugging.